ServerCore/Pages/Hints/CreateBulk.cshtml.cs

```diff
@@ -0,0 +1,138 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.EntityFrameworkCore;
+using ServerCore.DataModel;
+using ServerCore.ModelBases;
+
+namespace ServerCore.Pages.Hints
+{
+    [Authorize(Policy = "IsEventAdminOrAuthorOfPuzzle")]
+    public class CreateBulkModel : EventSpecificPageModel
+    {
+        public CreateBulkModel(PuzzleServerContext serverContext, UserManager<IdentityUser> userManager) : base(serverContext, userManager)
+        {
+        }
+
+        [BindProperty]
+        public bool DeleteExisting { get; set; }
+
+        [BindProperty]
```

Comment:
You might be able to add [Required] on these fields to have the user required to enter something and get rid of a couple of checks below

Reply:
The issue is that these fields are multiline and each line needs to be validated. [Required] isn't strong enough.